Apps/Contoso.Forms.Demo/Contoso.Forms.Demo/ModulePages/AppCenterContentPage.xaml.cs

```diff
@@ -7,13 +7,20 @@ namespace Contoso.Forms.Demo
     [Android.Runtime.Preserve(AllMembers = true)]
     public partial class AppCenterContentPage : ContentPage
     {
+
+        Fabulous.CustomControls.CustomEntryCell customEntryCell;
```

Comment:
Please do not use new third party libs just for that.
Plus it does not build:

<img width="516" alt="screen shot 2019-03-04 at 12 09 54 pm" src="https://user-images.githubusercontent.com/623672/53759989-79adb100-3e76-11e9-85ea-9b04fcf49d4e.png">

Reply:
Removed